src/web/routes/application/send-code/send-code.js

```diff
@@ -12,7 +15,13 @@ const pageContent = ({ translate }) => ({
 })
 
 const behaviourForPost = (req, res, next) => {
-  req.session[CONFIRMATION_CODE_SESSION_PROPERTY] = '123456'
+  if (validationResult(req).isEmpty()) {
+    // TODO DW HTBHF-1735 Randomly generate the code
+    const confirmationCode = '123456'
+    sendConfirmationCode(req.session.claim, req.body.channelForCode, confirmationCode)
+    req.session[CONFIRMATION_CODE_SESSION_PROPERTY] = '123456'
```

Comment:
I presume you should be using `confirmationCode` here.

Reply:
That constant is for `confirmationCode`.